### ISSUE TITLE
Error in Style/FirstParameterIndentation cop

### ISSUE THREAD

**Issue by atz** [bbatsov/rubocop#2341]
Interestingly, when I run `rubocop --auto-gen-config`, I get some errors:
```
2 errors occurred:
An error occurred while Style/FirstParameterIndentation cop was inspecting /Users/atz/repos/dor-services/lib/dor/datastreams/geo_metadata_ds.rb.
An error occurred while Style/FirstParameterIndentation cop was inspecting /Users/atz/repos/dor-services/lib/dor/datastreams/geo_metadata_ds.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.32.1 (using Parser 2.3.0.pre.2, running on ruby 1.9.3 x86_64-darwin13.3.0)
```

But if I run rubocop on that single file, it does not report any error.

Debug trace for above looks like:
```
An error occurred while Style/FirstParameterIndentation cop was inspecting /Users/atz/repos/dor-services/lib/dor/datastreams/geo_metadata_ds.rb.
undefined method `parent' for nil:NilClass
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/style/first_parameter_indentation.rb:42:in `message'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/cop.rb:162:in `add_offense'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/mixin/autocorrect_alignment.rb:27:in `block in check_alignment'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/mixin/autocorrect_alignment.rb:16:in `each'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/mixin/autocorrect_alignment.rb:16:in `check_alignment'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/style/first_parameter_indentation.rb:35:in `on_send'
(eval):5:in `block (2 levels) in on_send'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/commissioner.rb:92:in `with_cop_error_handling'
(eval):4:in `block in on_send'
(eval):2:in `each'
(eval):2:in `on_send'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:239:in `process'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:255:in `block in process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `map'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/parser-2.3.0.pre.2/lib/parser/ast/processor.rb:143:in `on_send'
(eval):9:in `on_send'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:239:in `process'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:255:in `block in process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `map'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/parser-2.3.0.pre.2/lib/parser/ast/processor.rb:143:in `on_send'
(eval):9:in `on_send'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:239:in `process'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/parser-2.3.0.pre.2/lib/parser/ast/processor.rb:81:in `on_casgn'
(eval):9:in `on_casgn'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:239:in `process'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:255:in `block in process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `map'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/parser-2.3.0.pre.2/lib/parser/ast/processor.rb:9:in `process_regular_node'
(eval):9:in `on_begin'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:239:in `process'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:255:in `block in process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `map'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/parser-2.3.0.pre.2/lib/parser/ast/processor.rb:9:in `process_regular_node'
(eval):9:in `on_class'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:239:in `process'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:255:in `block in process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `map'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/parser-2.3.0.pre.2/lib/parser/ast/processor.rb:9:in `process_regular_node'
(eval):9:in `on_module'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:239:in `process'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:255:in `block in process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `map'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:254:in `process_all'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/parser-2.3.0.pre.2/lib/parser/ast/processor.rb:9:in `process_regular_node'
(eval):9:in `on_begin'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/ast-2.0.0/lib/ast/processor.rb:239:in `process'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/commissioner.rb:52:in `investigate'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cop/team.rb:33:in `inspect_file'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:135:in `inspect_file'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:106:in `block in do_inspection_loop'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:100:in `loop'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:100:in `do_inspection_loop'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:78:in `process_file'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:53:in `block in inspect_files'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:51:in `each'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:51:in `inspect_files'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/runner.rb:30:in `run'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/lib/rubocop/cli.rb:26:in `run'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/bin/rubocop:13:in `block in <top (required)>'
/Users/atz/.rvm/rubies/ruby-1.9.3-p547/lib/ruby/1.9.1/benchmark.rb:295:in `realtime'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/gems/rubocop-0.32.1/bin/rubocop:12:in `<top (required)>'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/bin/rubocop:23:in `load'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/bin/rubocop:23:in `<main>'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/bin/ruby_executable_hooks:15:in `eval'
/Users/atz/.rvm/gems/ruby-1.9.3-p547/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by jonas054** [comment]
It would help to see the code that `rubocop` fails on. Is `geo_metadata_ds.rb` code you can share? If not, can you narrow down the problem to a minimal part of the code?

**Comment by alexdowad** [comment]
Upgrade the version of RuboCop you are running. 0.32.1 was released on June 24. A commit was added on July 21 which should fix the problem you are describing.

**Comment by alexdowad** [comment]
This can be closed.
